### ISSUE TITLE
raise-missing-from on bare raise

### ISSUE THREAD

**Issue by elelayan** [PyCQA/pylint#3790]
## disable raise-missing-from on re-raise?

Hi,
is **raise-missing-from** relevant when re-raising from an exception handler using a bare `raise`?

For instance in this kind of code:

```
try:
    myfunc()
except Exception:
    mark_failed()
    raise
```

Thanks,

**Comment by elelayan** [comment]
Well, the warning doesn't occur, actually.
Sorry for the noise.

```testraise.py
""" whatever """

def main():
    """ hello """
    try:
        1/0
    except Exception:
        print("OOPS")
        raise


if __name__ == '__main__':
    main()
```
